tests/vmi_lifecycle_test.go

```diff
@@ -904,7 +904,7 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 				config.CPUModel = defaultCPUModel
 				kvconfig.UpdateKubeVirtConfigValueAndWait(*config)
 
-				vmi := libvmifact.NewCirros()
+				vmi := libvmifact.NewAlpine()
```

Comment:
Nit: You can go as far as using `NewGuestless`

Reply:
NewGuestless is not working for s390x. I am currently working on to fix it. I have also started a discussion in this channel.
https://kubernetes.slack.com/archives/C0163DT0R8X/p1726468656344359